### ISSUE TITLE
Pion without default interceptors

### ISSUE THREAD

**Issue by sergystepanov** [giongto35/cloud-game#268]
Pion without default interceptors which are [NACK](https://github.com/pion/webrtc/blob/de913a99a885cf17d14a3ffa3de2f3b99039ad7d/interceptor.go), noticeably increases app responsiveness. 
https://github.com/giongto35/cloud-game/blob/77d7a7383feec8c14dc82dc4d1706de0870b620c/pkg/webrtc/connection.go#L15-L18

To research the necessity of such interceptors.

> NACK stands for Negative Acknowledgement. It is one of the error resiliency mechanisms in WebRTC.
> NACK is a way for the receiving end to indicate it hasn’t received a specific packet.
> A NACK message is sent over RTCP to the sender of the media, which in turn needs to decide if it will retransmit the lost packet based on its availability in its cache and its estimate to the usefulness of the retransmission (will it be possible to use it once received).

**Comment by giongto35** [comment]
What do you mean by app responsiveness? Do you mean the packet will be smoother with or without NACK? What experiment we can do here?

**Comment by sergystepanov** [comment]
To be clear, all my issues with a short description or an outline are meant to be a (personal) memo for _possible_ future experiments. And in case of confirmed usefulness of that, I'll write a normal description + implement the feature (:/
But since you're interested.
With removed NACK interceptor or without `RegisterDefaultInterceptors` call, subjectively I can tell that game controls fells more responsive to that point when I can't see any latency at all. Locally. Of course, as a downside, I noticed periodical small random audio artifacts here and there. Maybe it's worth researching and testing some more.
And as an objective e2e latency test for this kind of stuff, I came up with an idea (working with that canvas video mirroring thing) of some kind of emulated optical latency tester. Maybe you've seen devices attached to your monitor that measure latency between keypress (mouse click) and pixel change on the monitor. So it's the same idea but we measure time in the browser from a keypress until pixels change in some area of the video (where that canvas video mirroring will be useful), I think that how it works. There are some problems due to encoding artifacts but it's still doable, I think.
